server/channels/store/sqlstore/post_store.go

```diff
@@ -2292,8 +2292,79 @@ func (s *SqlPostStore) AnalyticsUserCountsWithPostsByDay(teamId string) (model.A
 	return rows, nil
 }
 
+func (s *SqlPostStore) countBotPostsByDay(teamID, startDay, endDay string) (model.AnalyticsRows, error) {
+	var query sq.SelectBuilder
+	if teamID != "" {
+		query = s.getQueryBuilder().
+			Select("TO_CHAR(day, 'YYYY-MM-DD') as Name, num as Value").
+			From("bot_posts_by_team_day").
+			Where(sq.Eq{"teamid": teamID})
+	} else {
+		query = s.getQueryBuilder().
+			Select("TO_CHAR(day, 'YYYY-MM-DD') as Name, COALESCE(SUM(num), 0) as Value").
+			From("bot_posts_by_team_day").
+			GroupBy("Name")
+	}
```

Comment:
Nit: why is the column called `Name`? It's just a formatted date, right?

Reply:
To make it scannable to `model.AnalyticsRows`

Reply:
Well, I meant the specific name `Name`, not why it has a name, which makes sense. Does it need to be `Name` specifically? It's just a bit confusing, 1/5, so up to you.

Reply:
It is called `Name` because it refers to the name of the metric. It is a bit confusing because we use to to signify single stats and also series metrics.

Reply:
Oooh, ok, got it now. Didn't know it had to be the same as other metrics.